airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/parquet_parser.py

```diff
@@ -23,6 +23,9 @@ class ParquetParser(FileTypeParser):
 
     ENCODING = None
 
+    def check_config(self, config: FileBasedStreamConfig) -> Tuple[bool, Optional[str]]:
+        return True, None
```

Comment:
Ditto comment above. A comment in a docstring would help readers/maintainers know if more should happen here in the future or if we simply don't have any validation to do.